platform_as_a_service.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Please move this file to `content/it/platform_as_a_service.md`, like #688. 😊
https://github.com/cncf/glossary/pull/688/files#diff-48a41d0d9d4d075bc34168a4632b490a11da0ce97194d39abd6af63347a6d96b

Reply:
Did I manage? 🧐